apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-item/index.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { applyFilters } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { Post } from '../../types';
+import './style.scss';
+
+interface NavItemProps {
+	item: Post;
+	postType: { slug: string };
+	selected: boolean;
+	statusLabel?: string;
+}
+
+export default function NavItem( { item, postType, selected, statusLabel }: NavItemProps ) {
+	const buttonClasses = classNames( 'wpcom-block-editor-nav-item', {
+		'is-selected': selected,
+	} );
+
+	const titleClasses = classNames( 'wpcom-block-editor-nav-item__title', {
+		'is-untitled': ! item.title?.raw,
+	} );
+
+	const defaultEditUrl = addQueryArgs( 'post.php', { post: item.id, action: 'edit' } );
+	const editUrl = applyFilters(
+		'a8c.WpcomBlockEditorNavSidebar.editPostUrl',
+		defaultEditUrl,
+		item.id,
+		postType.slug
+	);
+
+	return (
+		<li>
+			<Button
+				className={ buttonClasses }
+				href={ editUrl }
+				target={ applyFilters( 'a8c.WpcomBlockEditorNavSidebar.linkTarget', undefined ) }
+			>
+				<div className="wpcom-block-editor-nav-item__title-container">
+					<div className={ titleClasses }>
+						{ item.title?.raw || __( 'Untitled', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'Untitled', { context: 'post title for posts with no title'} )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/nav-sidebar-clickable-links)

Reply:
I've added a matching context string to this PR since it's the same. However the domain is different so not sure if it'll be able to match perfectly.